src/styles/_item-details.scss

```diff
@@ -93,4 +94,16 @@ img.backorder {
       }
     }
   }
+  .info{
+    position: absolute;
+    right: 10px;
+    margin: 0 10px;
+    padding: 0 5px 10px 5px;
+    width: 45%;
+    text-align: left;
+
```

Comment:
might be good to add `font-weight: 700; color: black; text-shadow 0 0 1px white` for readability